packages/registrar_client/lib/src/client.dart

```diff
@@ -14,6 +14,7 @@ class RegistrarClient {
   final String mnemonicOrSeed;
   final KPType keypairType;
 
+  late final int? twinId;
```

Comment:
- u can use a private field with a getter for twinId.
- Like this: 
```  
  int? _twinId;
  
  int? get twinId => _twinId;
```
we can also do the same for accountId, secretSeed, and privateKey